app/controllers/application_controller.rb

```diff
@@ -5,4 +5,11 @@ class ApplicationController < ActionController::Base
   respond_to :html
 
   protect_from_forgery with: :exception
+
```

Comment:
This method should be private so that it cannot be accessed externally

Reply:
Done in sha: cbdf00b